hbase-server/src/test/java/org/apache/hadoop/hbase/wal/TestWALFactory.java

```diff
@@ -676,12 +676,12 @@ public void visitLogEntryBeforeWrite(RegionInfo info, WALKey logKey, WALEdit log
       increments++;
     }
 
-    @Override
-    public void visitLogEntryBeforeWrite(WALKey logKey, WALEdit logEdit) {
-      // To change body of implemented methods use File | Settings | File
-      // Templates.
-      increments++;
-    }
+//    @Override
```

Comment:
Just remove this method?

Reply:
OK. A test legacy. Removed in the latest commit.